x-pack/plugins/security_solution/common/api/timeline/get_timelines/get_timelines_route.schema.yaml

```diff
@@ -64,24 +65,34 @@ paths:
         - in: query
           name: status
           schema:
-            allOf:
-             - $ref: '../model/components.yaml#/components/schemas/TimelineStatus'
-             - nullable: true
+            $ref: '../model/components.schema.yaml#/components/schemas/TimelineStatus'
+            nullable: true
       responses:
         '200':
           description: Indicates that the (template) timelines were found and returned.
           content:
             application/json:
               schema:
                 type: object
+                required: [data]
                 properties:
                   data:
                     type: object
+                    required:
+                      [
+                        timelines,
+                        totalCount,
+                        defaultTimelineCoun,
```

Comment:
ahh.. forgot a `t` here 😄

Reply:
Would be cool if the generator errored out if a field added here didn't actually exist in the schema